services/QuillLMS/app/assets/stylesheets/pages/premium_hub/snapshots.scss

```diff
@@ -0,0 +1,210 @@
+#premium-hub {
+  .snapshot-section {
+    box-shadow: 0px 0px 10px 0px rgba(0, 0, 0, 0.10);
+    border-radius: 8px;
+    padding: 24px;
+    border: 1px solid $quill-grey-5;
+    .snapshot-item {
+      background: $quill-grey-1;
+      border: 1px solid $quill-grey-5;
+      border-radius: 8px;
+      display: flex;
+      justify-content: space-between;
+      min-height: min-content;
+      position: relative;
+      h3 {
+        font-weight: 700;
+        font-size: 18px;
+        line-height: 22px;
+      }
+      .count-and-label {
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+      }
+      .count {
+        font-weight: 700;
+        font-size: 28px;
+        line-height: 34px;
+      }
+      .snapshot-label {
+        font-weight: 600;
+        color: $quill-grey-90;
+        .label-sub-text {
+          font-size: 14px;
+          font-style: normal;
+          font-weight: 400;
+          line-height: normal;
+          margin-top: 4px;
+          display: block;
+        }
+      }
+      .change {
+        font-weight: 700;
+        font-size: 20px;
+        line-height: 24px;
+        color: $quill-grey-90;
+        display: flex;
+        height: min-content;
+        align-items: center;
+        img {
+          margin-right: 4px;
+          &.small {
+            height: 26px;
+          }
+          &.medium {
+            height: 31px;
+          }
+        }
+      }
+      &.snapshot-count {
+        .loading-spinner-wrapper {
+          position: absolute;
+          left: 50%;
+          top: 50%;
+          transform: translate(-50%, -50%);
+        }
+      }
+      &.small {
+        padding: 16px;
+        height: 133px;
+        .snapshot-label {
+          font-size: 14px;
+          line-height: 18px;
+        }
+      }
+      &.medium {
+        padding: 32px;
+        height: 159px;
+        .snapshot-label {
+          font-size: 16px;
+          line-height: 21px;
+        }
+      }
+      &.positive, &.negative, &.no-change {
+        .count {
+          color: $quill-grey-90;
+        }
+      }
+      &.positive {
+        background: #F1FAF6;
+        border: 1px solid #B4E6D1;
+        .change {
+          color: #129E6C;
```

Comment:
shouldn't these be color variables? (or flagged to Jack for mapping?)

Reply:
Yes, you're right! This was just copied over from the `usage_snapshot_report` scss file since I wanted to make it usable outside of that report, and that scss was written before we had the variables in place. It doesn't look like these correspond to anything we have in the design files right now, so I will make a note to ask Jack about them in the new year.